### ISSUE TITLE
Hover callback won't fire

### ISSUE THREAD

**Issue by blazedd** [develersrl/angular-flot#31]
Angular v1.5.0-beta.2
See the flot graph usage here: https://github.com/blazedd/Minetrack/tree/angular

With scope set to `&` for `onPlotClick`, `onPlotHover`, & `onPlotSelected` you cannot specify a callback function from the scope. If set to `=` the function runs as expected.

```html
<flot height="100px" width="400px" dataset="[graph]" options="chartOpts" on-plot-hover="setTooltip" />
```

```javascript
minepack.controller('minepackHome', function($scope){
     $scope.graph = [[1, 2], [5, 10]];
     $scope.chartOpts = {...};

     // this will NOT fire
     $scope.setTooltip = function(data){
          console.log('fire event', data);
     };
});
```

If you check the flot code I confirm the event is firing, but is not calling a valid point of reference to my function. Am I being numbskull and expecting something different?

**Comment by lvillani** [comment]
Sorry for the delay, I'll try to investigate it this week.

**Comment by lvillani** [comment]
What happens if you try `on-plot-hover="setTooltip()"` instead of `on-plot-hover="setTooltip"`?

**Comment by lvillani** [comment]
Closing due to missing activity. Feel free to re-open it if necessary.

**Comment by maeishoj** [comment]
Hey there. I have the same issue! I can't seem to use the tooltip with angular flot chart.. 

I tried to add this to my config but no luck.

tooltip: {
            show: true,
            // content: "%s for %x was %y",
            // xDateFormat: "%y-%0m-%0d",
            onHover: function(flotItem, $tooltipEl) {
                console.log(flotItem, $tooltipEl);
            }
        }

I tried the approach above too again with no luck.

It works the pure javascript way with this example:

https://github.com/flot/flot/blob/master/examples/interacting/index.html


Any suggestion?
